source/Common/FrameLimiter.cs

```diff
@@ -4,15 +4,21 @@
 
 namespace Common
 {
+    /// <summary>
+    ///     Tries to achieve a minimum given frame time by waiting an appropriate amount of time each frame. Waiting
+    ///     is either done through a spin lock or sleep, depending on the amount of time that needs to be spent waiting.
+    /// </summary>
     public class FrameLimiter
     {
-        private readonly MovingAverage m_Avg;
-        private readonly long m_TargetTicksPerFrame;
-        private readonly Stopwatch m_Timer;
-        private readonly Action<long> m_WaitUntilTick;
-        public TimeSpan LastFrameTime;
-        private double m_AverageTicksPerFrame;
+        /// <summary>
+        ///     The elapsed time between the last call to <see cref="Throttle"/> and the call before that.
+        /// </summary>
+        public TimeSpan LastThrottledFrameTime;
```

Comment:
Might be worth giving something like this a private setter?
```suggestion
        public TimeSpan LastThrottledFrameTime { get; private set; }
```

Reply:
7128bad